### PR TITLE
capnp 0.5.1.2

### PR DESCRIPTION
**Pull request by DomT4**
Fixes 4 CVEs:

* CVE-2015-2310
* CVE-2015-2311
* CVE-2015-2312
* CVE-2015-2313

Security Advisory 1: [Clicky](https://capnproto.org/news/2015-03-02-security-advisory-and-integer-overflow-protection.html)

Security Advisory 2: [Clicky](https://capnproto.org/news/2015-03-05-another-cpu-amplification.html).

### CODE REVIEW

File: Library/Formula/capnp.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ def install
   end
 
   test do
-    system "#{bin}/capnp", "--version"
+    system bin/"capnp", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/capnp.rb]: Let's leave this line as-is.

Comment (DomT4) [inline Library/Formula/capnp.rb]: Whoops, Sorry. It's buried in my subconscious these days. Will undo.

**Comment by tamird** [comment]
:+1:

**Comment by mikemcquaid** [comment]
Thanks @DomT4!

**Pull request closed without merging**